.github/ISSUE_TEMPLATE/feature-request.yml

```
@@ -0,0 +1,35 @@
+name: Feature Request
+description: Suggest a new feature or an idea for ACCESS-Hive Docs.
+labels: ['new feature']
+body: 
+  - type: markdown
+    attributes: 
+      value: >-
+        Thanks for taking the time to suggest a new feature for ACCESS-Hive Docs!
+
+  - type: textarea
+    attributes:
+      label: Feature description
+      description: >-
+        Describe the feature you would like to see implemented. 
+        Feel free to include screenshots and examples if relevant.
+      placeholder: >-
+        Users currently struggle with... 
+        This new feature or idea would be... and would help with...
+    validations:
+        required: true
+
+  - type: textarea
+    attributes:
+      label: Possible Implementation
+      description: >-
+        (Optional) How could this feature be implemented? Please share any specific technical details or general high level suggestions.
```

Very minor, but more readable.
```suggestion
      description: >-
        (Optional) How could this feature be implemented? 
        Please share any specific technical details or general high level suggestions.
```